source/agora/consensus/data/Enrollment.d

```diff
@@ -51,6 +51,9 @@ public struct Enrollment
     /// The cycle length for a validator
     public static immutable uint ValidatorCycle = 1008; // freezing period / 2
 
+    /// The minimum number of validators required to a create block
+    public static immutable uint MinValidatorCount = 2;
```

Comment:
Why? This is introducing an arbitrary protocol change.

Reply:
Yes, you're right. I'll remove this.